tests/test_amin_amax.py

```diff
@@ -7,7 +7,7 @@
 from .helper import get_all_dtypes
```

Comment:
Could you please add `test_amin_amax.py` to test scope in `.github/workflows/conda-package.yml`?